src/main/java/org/wildfly/security/auth/realm/FileSystemSecurityRealm.java

```diff
@@ -347,20 +347,26 @@ public SupportLevel getEvidenceVerifySupport(final Class<? extends Evidence> evi
             List<Credential> credentials = loadCredentials();
             for (Credential credential : credentials) {
                 if (credential.canVerify(evidenceType, algorithmName)) {
+                    ElytronMessages.log.tracef("FileSystemSecurityRealm - evidence verification SUPPORTED: type = [%s]  algorithm = [%s]  credentials = [%d]", evidenceType, algorithmName, credentials.size());
```

Comment:
Hm maybe we should have multiple categories e.g. "org.wildfly.security.realm.filesystem"